src/components/settings/IntegrationsManager.tsx

```
@@ -0,0 +1,291 @@
+
+                            <div className="flex items-center gap-3 flex-shrink-0">
+                                <button
+                                    onClick={() => testIntegration(integration.id)}
+                                    disabled={isTesting === integration.id}
+                                    className="px-3 py-1.5 text-xs font-bold text-indigo-600 dark:text-indigo-400 bg-indigo-50 dark:bg-indigo-900/20 rounded-lg hover:bg-indigo-100 dark:hover:bg-indigo-900/40 disabled:opacity-50 transition-colors"
+                                >
+                                    {isTesting === integration.id ? 'Sending...' : 'Test Ping'}
+                                </button>
+
+                                {/* Toggle Switch */}
+                                <label className="relative inline-flex items-center cursor-pointer">
```

ESLint

[error] Delete `················`

(prettier/prettier)